### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by codepathreview** [kristianss27/NYTimesSearch#1]
@kristianss27 : Required story `User can tap on any article in results to view the contents in an embedded browser.` is not complete. Pls complete it asap and update submisison

**Comment by kristianss27** [comment]
I've add the behavior to open any article in a Web view. Thanks for letting me know. I totally forgot that required task. I did the change on my code adding it and changing the README with a new video. Would It be all right?

**Comment by nesquena** [comment]
Yes, please re-submit the project through the course portal when ready and we'll take another look. Be sure to add a video that includes the webview feature.

**Comment by kristianss27** [comment]
I did it already. I would appreciate to know Is it ok?

**Comment by kristianss27** [comment]
Thanks for giving time to resolve the problem. I'm willing to do anything necessary

**Comment by nesquena** [comment]
Graders will be looking again soon, they all have full-time jobs so harder for them to look during the day. They will make final evaluations. If we do accept the submission, can you commit to submitting all subsequent projects on time with required completed? We only grant a single extension per student on a case-by-case basis over 8 weeks. If you are concerned about being able to complete the projects on-time, better to withdraw now before being grouped.

**Comment by kristianss27** [comment]
Yes, I definitely can do it and I'm willing to do anything to continue working on the bootcamp. I would really appreciate if You can accept the last release I've sent. Thanks for giving time to resolve the problem. and once again I'm willing to do anything necessary

**Comment by codepathreview** [comment]
Decent work. A few notes after checking out the code:
- Nice to see you properly reused the single base method in search activity to fetch results for both initial load and the pagination.
- Nice to see you added the search box to the ActionBar!
- Nice to see you properly used spinners and checkboxes to allow users to select filters.
- Good to see you properly sent and received relevant filters.
- One thing to consider is rather than sending and returning all the filter values separately as keys, might have introduced a serializable model (i.e `SearchFilters.java`) representing all the filters.
- Consider using [GSON](https://guides.codepath.com/android/Leveraging-the-Gson-Library) to conveniently serialize and deserialize API responses into objects.
- Consider using a more robust HTTP client lib such as Retrofit or OkHttp. 

Also, here's a more detailed [Project 2 Feedback Guide here](http://courses.codepath.com/snippets/intro_to_android/project_2_feedback) which covers the most common issues with this submitted project. Read through the [feedback guide point-by-point](http://courses.codepath.com/snippets/intro_to_android/project_2_feedback) to determine how you could improve your submission.

Let us know if you have any other thoughts or questions about this assignment. The next assignment (Twitter Client) will be especially important since it introduces the majority of the remaining pieces necessary to build a fully functional API client with complex feeds of data and user creation.
